viewer/packages/cad-parsers/package.json

```diff
@@ -1,17 +1,18 @@
-{
-  "name": "@reveal/cad-parsers",
-  "private": true,
-  "main": "index.ts",
-  "scripts": {
-    "start": "yarn ws:start",
-    "test": "yarn ws:test --config ./../../jest.config.js",
-    "lint": "yarn ws:lint"
-  },
-  "dependencies": {
-    "@reveal/data-providers": "workspace:*",
-    "@reveal/metrics": "workspace:*",
-    "@reveal/model-base": "workspace:*",
-    "@reveal/sector-parser": "workspace:*",
-    "@reveal/utilities": "workspace:*"
-  }
-}
+{
+  "name": "@reveal/cad-parsers",
+  "private": true,
+  "main": "index.ts",
+  "scripts": {
+    "start": "yarn ws:start",
+    "test": "yarn ws:test --config ./../../jest.config.js",
+    "lint": "yarn ws:lint"
+  },
+  "dependencies": {
+    "@reveal/data-providers": "workspace:*",
+    "@reveal/logger": "workspace:*",
```

Comment:
Added a logger reference is the only change in this file except for line endings